apps/dot_ext/admin.py

```diff
@@ -21,8 +24,72 @@ class Meta:
         app_label = "bluebutton"
 
 
+class CustomAdminApplicationForm(CustomRegisterApplicationForm):
+
+    logo_image_bypass = BooleanField(label='Bypass Logo Image Upload Restrictions', required=False,
```

Comment:
Can this be simplified by allowing admins to set the `logo_uri` directly?

Reply:
The logo_uri field can be edited in the admin too. I thought this bypass might be useful to facilitate an upload of a non standard image file to S3, if needed.  However, I'm good with removing this which would simplify the form clean code.

Reply:
Removed logo_image_bypass.